map_creator/include/map_creator/kinematics.h

```diff
@@ -30,7 +30,7 @@ class Kinematics{
  float NORM(float a, float b, float c, float d);  
     void getPoseFromFK( const std::vector<double>  joint_values,  std::vector<double>& pose);
 
-    bool isIKSuccess(std::vector<double> pose,std::vector<double>& joints);
+    bool isIKSuccess(std::vector<double> pose,std::vector<double>& joints, int& numOfSolns );
```

Comment:
The header file was changed, but the .cpp file was not which results in a compile error.  Please commit any necessary changes to the .cpp file.